src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let countOfLoads = 0;
+
+  return (x) => {
+    countOfLoads++;
+
+    return (y) => {
+      return countOfLoads >= 3 && countOfLoads % 2 === 0
```

Comment:
```suggestion
      return countOfLoads > 3 && countOfLoads % 2 === 0
```
first three should work fine